matomo/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM matomo:4.5-apache
+ARG MATOMO_VERSION
+FROM matomo:$MATOMO_VERSION
```

Comment:
The downside of this approach is that you loose dependabot upgrade PRs

Reply:
ahh thats a nice point.. i will remove this idea.. 

(it was already in there and i just was making it work again)

Reply:
Yeah, I might have been the guilty one introducing this mechanism, before @dependabot was a thing..